inst/www/reactive-graph.html

```diff
@@ -622,15 +1372,21 @@
   enter: function(data) {
     var node = nodes[data.id];
     node.running = true;
+    node.start = data.time;
     pushUndo(function() {
       node.running = false;
+      node.start = null;
     });
   },
   exit: function(data) {
     var node = nodes[data.id];
     node.running = false;
+    var timeElapsed = (parseFloat(data.time) - parseFloat(node.start)) * 1000
+    var timeLabel = 'time elapsed: ' + (Math.round(timeElapsed * 1000) / 1000) + ' ms';
+    node.timeLabel = time ? timeLabel : '';
```

Comment:
Usually you want to keep the data in its "native" form until you're ready to display it (i.e. store the `timeElapsed` float on the node, not the fully formatted string). If we decide we want to show the number in a different color or weight than the "time elapsed" label, then we can do that easily.